js/openfisca.js

```diff
@@ -29,6 +29,14 @@ function serialize(form) {
 		if (element.name == 'effectif_entreprise')
 			value ++
 
+		/* In the case of a `temps partiel`, we are asking hours per week,
+		the most common way to reason about it. But OpenFisca needs hours per month */
+		if (element.name == 'heures_remunerees_volume') {
+			var duree_legale_mensuelle = 151.66,
+				duree_legale_hebdomadaire = 35
```

Comment:
Because these are constants defined outside from the scope of this application. It is not a matter of code optimisation (even though one might argue minimising the amount of assignments would be a good thing), it is a matter of conceptual cleanliness. There is no way these values can vary from one invocation to the other, so they are not `var`iables from this scope. You can see this as a kind of IoC if it makes more sense for you.